Storage/SQL/JoinedHistoryVisitsTable.swift

```diff
@@ -150,13 +160,15 @@ class JoinedHistoryVisitsTable: Table {
         sql += "GROUP BY historyId";
 
         // Trying to do this in one line (i.e. options?.sort == .LastVisit) breaks the Swift compiler
+        println("Sort: \(options?.sort)")
```

Comment:
Maybe use XCGLogger instead, or just drop this line, so we don't pollute the Xcode logger (I noticed it's starting to fill up more, and it's getting harder to find things).